### DISCUSSION TITLE
hx-request , timeout setting give back an error

### DISCUSSION THREAD

**Discussion by BobSquarePants** [bigskysoftware/htmx#1381] [Q&A] [answered]
Hi everyone,

When I try 
```html
<div ... hx-request='\"timeout\":100'>
  ...
</div>
```
like mentioned on this page https://htmx.org/attributes/hx-request/

I got in return:
> SyntaxError: JSON.parse: expected property name or '}' at line 1 column 2 of the JSON data

So how. should it be writed ? Thanks

**Comment by BobSquarePants** [comment]
I just tried also
```html
<div ... hx-request="{'timeout':1000}">
  ...
</div>
```
But that give the same error.

**Comment by RoToRa** [comment, marked as answer]
The example looks wrong there. There shouldn't be backslashes in there. And single quotes are invalid in JSON. Try

    hx-request='"timeout":100'

or 

    hx-request='{"timeout":100}'

**Reply by BobSquarePants** [reply]
thank you @RoToRa I've tried the second one and it works 👍
